src/Authenticator/FormAuthenticator.php

```diff
@@ -89,10 +90,17 @@ protected function _buildLoginUrlErrorResult(ServerRequestInterface $request): R
             $uri = $uri->withPath((string)$base . $uri->getPath());
         }
 
+        $checkFullUrl = Hash::get((array)$this->getConfig('urlChecker'), 'checkFullUrl', false);
```

Comment:
Why don't you use dot notation to access nested config? `urlChecker.checkFullUrl` ?

Reply:
Because I'm a muffin, I totally forgot this was a thing.